draft-ietf-teep-protocol.md

```diff
@@ -86,6 +86,11 @@ normative:
   I-D.ietf-suit-manifest: 
   I-D.moran-suit-trust-domains: 
   I-D.moran-suit-report: 
+  COSE.Algorithm:
+    title: "CBOR Object Signing and Encryption (COSE)"
+    author:
+      org: IANA
+    target: https://www.iana.org/assignments/cose/cose.xhtml
```

Comment:
Shouldn't this point to https://www.iana.org/assignments/cose/cose.xhtml#algorithms specifically?
with title "COSE Algorithms" instead of "CBOR Object Signing and Encryption (COSE)"?

Reply:
Thanks for reviewing. I didn't know whether the fragment URI can be used to indicate the reference. Directly indicating the section of COSE algorithms registry is better.  I agree with your proposal.